mailet/icalendar/src/main/java/org/apache/james/transport/mailets/ICALToJsonAttribute.java

```diff
@@ -82,14 +85,26 @@
  * </pre>
  */
 public class ICALToJsonAttribute extends GenericMailet {
+    @SuppressWarnings("unchecked")
+    private static final Class<Map<String, Calendar>> MAP_CALENDAR_CLASS = (Class<Map<String, Calendar>>)(Object) Map.class;
```

Comment:
Maybe we can share these constants in an API?

Reply:
I'm not convinced, it's for reading purposes initially, it's very specific I think.